browser/ui/views/brave_ads/ad_notification_view.cc

```diff
@@ -97,8 +107,11 @@ void AdNotificationView::OnMouseReleased(const ui::MouseEvent& event) {
     return;
   }
 
-  const std::string id = ad_notification_.id();
-  AdNotificationPopup::OnClick(id);
```

Comment:
`AdNotificationPopup::OnClick` contained:
- `AdNotificationPopup::FadeOut` which leads to popup close after some time
- AdNotificationDelegate::OnClick, which leads to another call by following chain:
    AdEventClicked::FireEvent() -> AdsClientHelper::Get()->CloseNotification() -> AdNotificationPopupHandler::Close()

To avoid this I have removed explicit popup close in `AdNotificationView::OnMouseReleased()`